_data/events.yml

```diff
@@ -0,0 +1,30 @@
+- title: |
```

Comment:
I think this event it is also in announcements, maybe is better to remove from there to not be duplicated in home page.